spec/acceptance/nodesets/default.yml

```diff
@@ -9,6 +9,8 @@ HOSTS:
   el7:
     roles:
       - server-el7
+      # roles migrated from now-removed el6 node(s): 
+      - server-el6
```

Comment:
server-el6 is not appropriate.  Please remove.

Reply:
Automation updated & removed